### DISCUSSION TITLE
Is it possible to limit what articles are imported to Obsidian by label?

### DISCUSSION THREAD

**Discussion by Marcel7Be4** [omnivore-app/omnivore#3379] [Q&A]
I use the Obsidian plugin to export articles, but not all of them are of use to me there. Is it possible to specify what gets does not get imported? Newsletters, for example.

**Comment by jacksonh** [comment]
Hey @Marcel7Be4 yeah you can do this with a custom query in the plugin settings, a common one people use is `in:all label:"Send to Obsidian"` and then they create a label with that name. You could also only have it sync after being archived by doing something like: `in:archive label:"Send to Obsidian"`
